awesomplete.js

```diff
@@ -301,9 +309,7 @@ function $$(expr, con) {
 	return slice.call((con || document).querySelectorAll(expr));
 }
 
-$.create = function(tag, o) {
-	var element = document.createElement(tag);
-
+$.set = function(element, o) {
```

Comment:
I did release the helpers: https://github.com/leaverou/bliss
Not sure what you mean with the rest of your comment. Are you referring to adding gulp? In any case, PRs are welcome :)